docs/guide/features.md

```diff
@@ -44,16 +44,16 @@ Learn more about [Test Filtering](./filtering.md).
 
 ## Running tests concurrently
 
-Use `.concurrent` in consecutive tests to run them in parallel.
+Use `.concurrent` in consecutive tests to run them in parallel. Also, be sure to use `expect` from the [Test Context](/guide/test-context.md) to ensure the right test is being detected.
```

Comment:
I think we should add `::warning` section and mention snapshot testing there and `expect.assertions`. Otherwise, it doesn't matter what expect you use there.